### PR TITLE
[MongoDB] Extended the guide

### PR DESCRIPTION
**Pull request by kastnerp**
Added instructions to access MongoDB externally without an SSH tunnel

### CODE REVIEW

**Review by nichtmax — CHANGES_REQUESTED** [review]
Thanks for your addition! From a security perspective you should only open ports with encryption. [MongoDB supports TLS](https://www.mongodb.com/docs/manual/tutorial/configure-ssl/), [we provide the certificates](https://manual.uberspace.de/web-https/#certificate-access). As we have a teaching assignment here could you please update your PR with TLS configuration?

File: source/guide_mongodb.rst [hunk 1]
```diff
@@ -157,6 +157,58 @@ Now you can just run ``mongo`` to connect to your MongoDB instance:
...
+   --auth
+   --unixSocketPrefix %(ENV_HOME)s/mongodb
+   --tlsMode requireTLS 
+   --sslPEMKeyFile %(ENV_HOME)s/mongodb/%(ENV_USER)s.uber.space.pem
+ autostart=yes
+ autorestart=yes
+ startsecs=30
+
+
+Create the `.pem` file with the certificate chain:
```
Comment (SalocinHB) [inline source/guide_mongodb.rst:191]: The following steps need to be repeated whenever the Let's Encrypt certificate changes (typically every 60 days).

File: source/guide_mongodb.rst [hunk 2]
```diff
@@ -157,6 +157,58 @@ Now you can just run ``mongo`` to connect to your MongoDB instance:
...
+ [isabell@stardust ~]$ 
+
+Update and restart the service.
+
+.. code-block:: none
+
+ [isabell@stardust ~]$ supervisorctl stop mongodb
+ supervisorctl reread 
+ supervisorctl update mongodb 
+ supervisorctl start mongodb
```
Comment (SalocinHB) [inline source/guide_mongodb.rst:205]: This should be a single `supervisorctl restart mongodb`. If the service's `.ini` hasn't changed, `reread` and `update` won't do anything and `start` would only tell you that it's already running.

File: source/guide_mongodb.rst (hunk 1 above)
Comment (SalocinHB) [inline source/guide_mongodb.rst:191]: The following steps need to be repeated whenever the Let's Encrypt certificate changes (typically every 60 days).

File: source/guide_mongodb.rst [hunk 3]
```diff
@@ -157,6 +157,58 @@ Now you can just run ``mongo`` to connect to your MongoDB instance:
...
+
+Update and restart the service.
+
+.. code-block:: none
+
+ [isabell@stardust ~]$ supervisorctl stop mongodb
+ supervisorctl reread 
+ supervisorctl update mongodb 
+ supervisorctl start mongodb 
+ [isabell@stardust ~]$
```
Comment (SalocinHB) [inline source/guide_mongodb.rst:206]: If only the certificate changed but the `.ini` remains the same, you also need a `restart`  here.

**Review by SalocinHB — CHANGES_REQUESTED** [review]
Please include a way to detect renewed certificates, that should trigger a replace of the `.pem` file and restart the service.

File: source/guide_mongodb.rst (hunk 1 above)
Comment (kastnerp) [reply source/guide_mongodb.rst:191]: Sure, how would you like me to do that? Any suggestions?

Comment (SalocinHB) [reply source/guide_mongodb.rst:191]: Periodically re-create the .pem file and restart the service. A `@monthly` cronjob springs to mind.

**Review by ankit-gautam23 — COMMENTED** [review]
Creating a cron job could be better option though.

**Comment by noave** [comment]
hey @kastnerp , we will take over the next days and complete the last issue (looks like its only the cronjob) here to complete the merge. If you wanna finish it yourself just give us a short notice :-)

**Comment by kastnerp** [comment]
Busy with work currently, please go ahead with adding the cronjob. Thank you! 🙂

**Comment by nichtmax** [comment]
@kastnerp please get in touch via hallo@uberspace.de to get your mug ☺️

**Pull request merged**